backend/app/Services/Quality/QualityTriggerService.php

```
@@ -0,0 +1,354 @@
+        }
+
+        $created = 0;
+        $batches = Batch::where('status', Batch::STATUS_IN_PROGRESS)->with('workOrder')->get();
+
+        foreach ($triggers as $trigger) {
+            foreach ($batches as $batch) {
+                if (! $trigger->matchesBatch($batch)) {
+                    continue;
+                }
+
+                $lastFiredAt = QualityControlTask::where('quality_control_trigger_id', $trigger->id)
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\QualityControlTask::where().


(staticMethod.notFound)